### ISSUE TITLE
highcharts-gauge says it isn't installed

### ISSUE THREAD

**Issue by leonsp** [MaazAli/highcharts-gauge#2]
I have maazalik:highcharts-gauge 0.3.1 installed, but I get this error in my browser's console when I try to use it:

```
maazalik:highcharts version 0.3.0 doesn´t support solid gauges anymore! Please use maazalik:highcharts-gauge in addition. maazalik_highcharts.js:21368:5
```

My Meteor package list:

```
$ meteor list
accounts-password                    1.1.1  Password support for accounts
autopublish                          1.0.3  Publish the entire database to all clients
email                                1.0.6  Send email messages
fezvrasta:bootstrap-material-design  0.3.0  FezVrasta's Bootstrap theme implementing Google's Material (Paper) Design
ian:accounts-ui-bootstrap-3          1.2.77  Bootstrap-styled accounts-ui with multi-language support.
insecure                             1.0.3  Allow all database writes by default
iron:router                          1.0.9  Routing specifically designed for Meteor
maazalik:highcharts-gauge            0.3.1  HighCharts gauge for Meteor
meteor-platform                      1.2.2  Include a standard set of Meteor packages in your app
mquandalle:harmony                   1.3.79  JavaScript.next-to-JavaScript-of-today compiler
stevezhu:lodash                      3.10.1  A utility library delivering consistency, customization, performance, & extras.
twbs:bootstrap                       3.3.5  The most popular front-end framework for developing responsive, mobile first projects on the web.
zimme:active-route                   2.3.1  Active route helpers
```

**Comment by jhuenges** [comment]
Found the mistake. `maazalik:highcharts-gauge` still uses `maazalik:highcharts@0.3.0`. A temporary fix might be to add `maazalik:highcharts@0.4.0` by hand. But I am not sure if this helps. I ll update the code. @MaazAli needs to update the package afterwards.

**Comment by MaazAli** [comment]
Pushed the changes to atmosphere.
